libraries/chain/block_state.cpp

```diff
@@ -45,13 +50,28 @@ block_state::block_state(const block_header_state& bhs, deque<transaction_metada
    sign(signer, valid_block_signing_authority);
 }
 
-// Used for transition from dpos to instant-finality
-block_state::block_state(const block_state_legacy& bsp) {
+// Used for transition from dpos to Savanna.
+block_state::block_state(const block_state_legacy& bsp, const digest_type& action_mroot_svnn) {
```

Comment:
Not sure why, but I don't like the `_svnn`  suffix at all. Maybe because it is all consonnants?

Reply:
`_savanna` is too long while `_s` is too short. Plus some use `_s` to indicate a struct. I think `_svnn` is a good compromise.

Reply:
While I agree that `_savanna` is too long, I guess we can agree to disagree on `_svnn` being a good compromise.